fggs/formats.py

```diff
@@ -316,7 +352,11 @@ def visit(d):
             if pos.startswith('"') and pos.endswith('"'):
                 pos = pos[1:-1]
             x, y = pos.split(',', 1)
-            positions[v.get_name()] = (float(x), float(y))
+            if re.match(r'"(.*)"', v.get_name()):
```

Comment:
Why not replace these 4 lines with just the 2nd line?

Reply:
This has been changed to one line.